htdocs/langs/en_US/cashdesk.lang

```diff
@@ -145,3 +145,6 @@ ShowProductReference=Show reference of products
 UsePriceHT=Use price excl. taxes and not price incl. taxes
 TerminalName=Terminal %s
 TerminalNameDesc=Terminal name
+DefaultPOSThirdLabel=TakePOS sales
```

Comment:
I suggest to name it "Default customer for POS"

Reply:
Approved for me